config/default.schema.json

```diff
@@ -564,6 +569,16 @@
 			"type": "string",
 			"description": "Password for the antivirus service, only used for legacy file implementation"
 		},
+		"CLAMAV_SERVICE_HOSTNAME": {
```

Comment:
Die beiden machen nur zusammen sinn. Daher würde ich sie in den Namespace CLAMAV packen und SERVICE_HOSTNAME und SERVICE_PORT dort einsortieren.